Packs/Zoom/Integrations/Zoom/Zoom.py

```diff
@@ -141,13 +144,14 @@ def get_oauth_token(self, force_gen_new_token=False):
                 # token expired
                 oauth_token = self.generate_oauth_token()
 
-        ctx.update({'token_info': {'oauth_token': oauth_token, 'generation_time': now.strftime("%Y-%m-%dT%H:%M:%S")}})
+        ctx.update({'token_info': {'oauth_token': oauth_token,
+                   'generation_time': now.strftime("%Y-%m-%dT%H:%M:%S")}})
         set_integration_context(ctx)
         return oauth_token
 
     def error_handled_http_request(self, method, url_suffix='', full_url=None, headers=None,
                                    auth=None, json_data=None, params=None,
-                                   return_empty_response: bool = False, resp_type: str = 'json'):
+                                   return_empty_response: bool = False, resp_type: str = 'json', stream: bool = False):
```

Comment:
What is the "stream" arg? Lets give it a meaningful name.

Reply:
thats the original name of the argument in the http_request, all i did is to pick out only the argument im using. so i think the best is to keep the name.